server/models.py

```diff
@@ -0,0 +1,10 @@
+from app import db
+from sqlalchemy.dialects.postgresql import JSON
+
+class User(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(80), nullable=False)
+    email = db.Column(db.String(80), nullable=False, unique=True)
```

Comment:
Small suggest to bump name and email length to 255 or somesuch and use that as the default string length.
But I'm fine with the rest of it so you can merge whenever.

Reply:
@thaumasiotes will be working with the User model on issue #14.

@hoodedice are you more comfortable with defining every string data to have 255 character limit? We need to sync this between the frontend and the backend to reduce conflict on error handling.

Reply:
Yes, I would like to have a greater character limit.